### ISSUE TITLE
Option to group tabs or not

### ISSUE THREAD

**Issue by adamjamesadair** [adamjamesadair/manage-my-tabs#133]
Add an option (toggle) to group tabs or not. Feature suggested in #130.

**Comment by BorFour** [comment]
I would suggest to add different sorting criteria to group the tabs, including "None" as a possibility (but this would be the same as having "Max tabs per tab group" = 1?

**Comment by rayman89** [comment]
Max tabs per tab group" = 1 keeps the groups and 1 tab below which occupies more space than needed and makes the UI not very clean for tab managment IMO.

**Comment by BorFour** [comment]
@rayman89 yeah, that is true. Maybe we could remove the tab groups when the max tabs per group is 1.
